pkg/utils/utils.go

```diff
@@ -524,7 +524,7 @@ func CreateProcessTree(process *apitypes.Process, shimPid uint32) (*apitypes.Pro
 	}
 
 	// build the process tree
-	treeRoot, err := buildProcessTree(proc, &procfs, shimPid, *process)
+	treeRoot, err := buildProcessTree(proc, &procfs, shimPid, nil)
```

Comment:
Are you sure about the nil?

Reply:
Yes, we the recursive call starts with a an empty process tree. Before this fix we had a duplicate parent-child entry